open-reac/src/main/java/com/powsybl/openreac/parameters/input/OpenReacParameters.java

```diff
@@ -39,10 +37,22 @@ public class OpenReacParameters {
 
     private final List<OpenReacAlgoParam> algorithmParams = new ArrayList<>();
 
+    // Algo parameters
+
     private OpenReacOptimisationObjective objective = OpenReacOptimisationObjective.MIN_GENERATION;
 
+    private static final String OBJECTIVE_DISTANCE_KEY = "ratio_voltage_target";
+
     private Double objectiveDistance;
 
+    private static final String MIN_VOLTAGE_LIMIT_CONSISTENCY_KEY = "consistent_min_voltage";
```

Comment:
Maybe `MIN_PLAUSIBLE_LOW_VOLTAGE_LIMIT` and `MAX_PLAUSIBLE_LOW_VOLTAGE_LIMIT`. Is is more coherent with choices already made for open loadflow.